DSharpPlus/Entities/DiscordUser.cs

```diff
@@ -84,6 +85,12 @@ public string DefaultAvatarUrl
         [JsonProperty("email", NullValueHandling = NullValueHandling.Ignore)]
         public virtual string Email { get; internal set; }
 
+        /// <summary>
+        /// Gets the user's email address.
```

Comment:
If you're doing that, and generally you shouldn't mix changes within PRs, please fix the docstring.